packages/web3-providers-ws/test/unit/constructor.test.ts

```diff
@@ -0,0 +1,10 @@
+import Web3ProviderWS from '../../src/index';
+import { WebSocketOptions } from '../../src/types';
+
+describe('Web3ProviderWS Tests', () => {
+    it('should error because invalid WS address', () => {
```

Comment:
We should have a valid instantiation test as well for sanity (even though it's technically tested with other tests)

Reply:
As discussed in call tests will be added in new PR (TODO).

Reply:
Created an issue to track this #4256